src/main/java/com/google/cloud/discotoproto3converter/disco/Method.java

```diff
@@ -95,7 +98,7 @@ public static Method from(DiscoveryNode root, Node parent) {
             path,
             pathParams,
             queryParams,
-            requiredParams,
+            requiredParamNames,
```

Comment:
(separately, not blocking) I'd be curious to talk about this `AutoValue_Method` and how it's used.

Reply:
I'm not sure what you mean by that. `AutoValue_` classes are just automatically generated classes. They are generated by autovalue plugin to javac for all classes marked with `@AutoValue` annotation. It is simply a POJO class with properties setters and getters, and it is pretty much it.

Reply:
Ah, OK. This is the standard pattern I've seen elsewhere. Never mind, then.

(For some reason on my first read I misinterpreted `Method` as referring to some sort of `@AutoValue` internal thing you were hacking, but it's not. It's just referring to an API method. My mistake!)